SDWebImage/SDWebImageWebPCoder.m

```diff
@@ -69,6 +70,18 @@ - (UIImage *)decodedImageWithData:(NSData *)data {
     }
     
     uint32_t flags = WebPDemuxGetI(demuxer, WEBP_FF_FORMAT_FLAGS);
+    
+    CGColorSpaceRef colorSpace = [self sd_colorSpaceWithDemuxer:demuxer];
+    
+    if (!(flags & ANIMATION_FLAG)) {
+        // for static single webp image
+        UIImage *staticImage = [self sd_rawWebpImageWithData:webpData colorSpace:colorSpace];
+        WebPDemuxDelete(demuxer);
```

Comment:
Why did you give up draw on CGBitmapContext?

Reply:
See the comment below, if you don't understand both "Why previously we draw on bitmap context and grab again ?", and "Why we currently don't need draw on bitmap context and grab again ?"